website/public/category-descriptions/vertebrates-genome-project.md

```diff
@@ -1,3 +1,29 @@
-A comprehensive suite of genomic workflows developed for generating reference-quality vertebrate genome sequences. The framework features specialized methods for high-quality genome assembly, haplotype phasing, and chromosome-scale scaffolding. The VGP pipeline offers flexible assembly trajectories based on available data types (PacBio HiFi solo, trio with parental Illumina, or HiFi with HiC phasing) with optional scaffolding using Bionano optical maps and/or Hi-C data.
+A comprehensive suite of genomic workflows developed for generating reference-quality vertebrate genome sequences with PacBio HiFi sequencing data. While optimised for vertebrate data, they are suitable for any diploid organism, but some parameter adjustments may be necessary. The framework features specialized methods for high-quality genome assembly, haplotype phasing, and chromosome-scale scaffolding. The VGP pipeline offers flexible assembly trajectories based on available data types (see below) with optional scaffolding using Bionano optical maps and/or Hi-C data.
```

Comment:
That should be a relatively short description and not a full-size summary. It would currently be more important to update the annotations for those workflows that are blank.

Reply:
Did you look at how that renders on the site ? If it looks nice maybe that's ok.